native/cocos/bindings/jswrapper/v8/Class.cpp

```diff
@@ -79,14 +79,14 @@ Class *Class::create(const ccstd::string &clsName, Object *parent, Object *paren
 
 Class *Class::create(const std::initializer_list<const char *> &classPath, Object *parent, Object *parentProto, v8::FunctionCallback ctor, void *data) {
     se::AutoHandleScope scope;
-    se::Object *currentParent = parent;
```

Comment:
If using `se::Object *currentParent` and `initializer_list` is more than 2, `currentParent` pointer in for loop will be an invalid pointer when `i >=1` .